1209/EBA7/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: USBscope
+owner: Victor Grigoryev
```

Comment:
The 'owner' field needs to match the directory name of the directory in which your org file is ('USBscope').